cdc/sink/mysql.go

```diff
@@ -88,17 +76,23 @@ func configureSinkURI(sinkURI string) (string, error) {
 	return dsnCfg.FormatDSN(), nil
 }
 
-// NewMySQLSinkUsingSchema creates a new MySQL sink
-func NewMySQLSinkUsingSchema(db *sql.DB, schemaStorage *schema.Storage) Sink {
-	inspector := &cachedInspector{
-		db:    db,
-		cache: make(map[string]*tableInfo),
-		tableGetter: func(_ *sql.DB, schemaName string, tableName string) (*tableInfo, error) {
-			info, err := getTableInfoFromSchemaStorage(schemaStorage, schemaName, tableName)
+// NewMySQLSinkUsingSchema creates a new MySQL sink using schema storage
+func NewMySQLSinkUsingSchema(sinkURI string, infoGetter TableInfoGetter, _ map[string]string) (Sink, error) {
```

Comment:
I think we should keep the name `NewMySQLSink` instead.

The name for the unused parameter should be kept, because otherwise we might forget what it's supposed to be used for (or we might just get rid of it).